docs/how-to-guides/faq.md

```diff
@@ -82,15 +82,15 @@ Currently, promptflow supports the following environment variables:
 
 **PF_WORKER_COUNT** 
 
-Valid for batch run only. The number of workers to use for parallel execution of the Flow.
+Valid for batch run only. The number of workers determine the degree of parallelism of the batch run execution.
 
-Default value is 16. If you have large number of batch run date row count, and want more efficiency, you can increase the PF_WORKER_COUNT to improve the batch run concurrency, make it run faster.
+The default value is 4. For large number data rows of batch run, if you want to improve the efficiency, consider increasing the PF_WORKER_COUNT to improve the concurrency.
```

Comment:
"For large number data rows of batch run, if you want to improve the efficiency, consider increasing the PF_WORKER_COUNT to improve the concurrency."
This sentence is unnecessary, it can be merged with "But there are two points to note"

Please consider changing to:
"Please take the following points into consideration when changing it:"